common/changes/@snowplow/node-tracker/feature-1178-nodejs-allow-setting-sessionid_2023-05-23-09-45.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@snowplow/node-tracker",
+      "comment": "Add setSessionId capability",
```

Comment:
```suggestion
      "comment": "Add setSessionId capability to Node.js tracker",
```

Reply:
Isn't this already built in based on the `packageName` theoretically ?

Reply:
Oh now I realize that the message relates directly to the `@snowplow/node-tracker` package, so the extra info is not important. So feel free to ignore.